po/th.po

```diff
@@ -0,0 +1,166 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR Endless Mobile, Inc.
+# This file is distributed under the same license as the eos-knowledge package.
+#
+# Translators:
+# Philip Chimento <philip.chimento@gmail.com>, 2017
+msgid ""
+msgstr ""
+"Project-Id-Version: eos-knowledge-lib\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-04-03 09:58+0800\n"
+"PO-Revision-Date: 2018-03-13 18:24+0000\n"
+"Last-Translator: Roddy Shuler <roddy@endlessm.com>\n"
+"Language-Team: Thai (http://www.transifex.com/endless-os/eos-knowledge-lib/"
+"language/th/)\n"
+"Language: th\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: eknrenderer/eknr-renderer.c:458
+#, fuzzy, c-format
+msgid "This page contains content from %s, available under a %s license."
+msgstr ""
+"หน้านี้ประกอบด้วยเนื้อหาจาก {original-link} ใช้งานได้ภายใต้ลิขสิทธิ์ของ {license-link}"
+
+#: eknrenderer/eknr-renderer.c:468
+msgid "http://wikihow.com"
+msgstr "http://wikihow.com"
+
+#: eknrenderer/eknr-renderer.c:470
+#, fuzzy, c-format
+msgid ""
+"See %s for more details, videos, pictures and attribution. Courtesy of %s, "
+"where anyone can easily learn how to do anything."
+msgstr ""
+"ดูรายละเอียด, วิดีโอ, ภาพ, และความเป็นเจ้าของเพิ่มเติมที่ {wikihow-article-link}  "
```

Comment:
This isn't updated to use the new `%s` format required for formatting the string in C. It will have undefined behaviour at runtime. Seems to be the same across all the languages that have translations for this string.

Reply:
Hrm, okay. This is quite surprising, since I explicitly regenerated the translations using `ninja eknr-update-po`. That should have rewritten all the translation files and removed the translations for non-matching strings.

I'll take another look.

Unfortunately, testing this is still somewhat of a pain since all those apps were built for SDK1, which doesn't use this library. But I might be able to test it by initializing getttext on the companion app side and setting a language there ... . I'll have to see.

Reply:
Yes, but you changed the source strings from `{wikihow-article-link}` to `%s`... so the translated strings also have to make the same change.

Reply:
Yeah, I figured that was the issue once I took a deeper dive into the problem.